src/events/components/ImageView/image.less

```diff
@@ -22,7 +22,7 @@
   margin-top: 20px;
 }
 
-@media (min-width: @owTabletBreakpoint) {
+@media (min-width: 1025px) {
```

Comment:
You should be able to use `@owTabletBreakpoint + 1` or similar with less if that's the intended breakpoint

Reply:
There are also two other breakpoints (600px and 320px) that are not attached to a variable atm :)

Reply:
Changed to `(min-width: @owTabletBreakpoint + 1)` as you suggested. The breakpoints you mentioned are specific to this page, so I don't see why they should/would be added to the variables we currently have.